src/components/controls/EditControls.jsx

```diff
@@ -0,0 +1,46 @@
+import React, { useState } from "react";
+
+function EditControls({item, onChange, onDelete}) {
+    const[displayUpdate, setDisplayUpdate] = useState(false)
+    return (
+        <>
+        <div>
+            <input 
+                type='checkbox' 
+                value={item.done} 
+                onChange={(e) => 
+                    {onChange({...item, done: e.target.checked})

```

Comment:
Struggled for a bit with the handler for this one but it was because I wasn't passing the item prop to a separate component. I believe I initially had it as handleUpdate. Anyhow yes, bad practice, won't be doing that in the future.

Reply:
You're not the only one that struggled with this one. The naming convention of props is one of the most difficult aspects of creating your app. Overall, you did really great! Just something to keep in mind when working on future projects. :)